### PR TITLE
Update API Key instructions

### PR DESCRIPTION
**Pull request by dfeinzimer**
swift/5514

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,9 +47,11 @@ The *ArcGIS Maps SDK for Swift Toolkit* has a *Target SDK* version of *16.0*, me
 
  New to Swift Package Manager? Visit [swift.org/package-manager/](https://swift.org/package-manager/).
 
-## Configure API Key
+## Configure API Key & Licensing
 
-Some of the toolkit components and examples utilize a set of ready-to-use ArcGIS Platform services, including basemaps, and therefore require an API Key to be set in `ExamplesApp.swift`. Please see the [setup guide](https://developers.arcgis.com/swift/get-started/#3-access-services-and-content-with-an-api-key/) for more information.
+Use of ArcGIS location services, such as basemaps, geocoding, and routing, requires either user authentication or API key authentication. Some of the toolkit components and examples utilize a set of these ready-to-use ArcGIS location services, including basemaps, and therefore require an API Key to be set in `ExamplesApp.swift`. Please see the [setup guide](https://developers.arcgis.com/swift/get-started/#3-access-services-and-content-with-an-api-key/), https://links.esri.com/arcgis-maps-sdk-security-auth and https://links.esri.com/arcgis-maps-sdk-license-and-deploy for more information.
```
Comment (dfeinzimer) [inline README.md:52]: 1. This is a slight modification to the suggested text to include the pre-existing "Some of the toolkit components..." language.
2. The 2nd and 3rd links can be updated to display a title `[]()` once those webpages are live and we have a good title to use.
3. This assumes the cutoff suggestion meant to include the 2nd link in the referenced issue (https://links.esri.com/arcgis-maps-sdk-license-and-deploy).

**Comment by dfeinzimer** [comment]
Closing for time being

File: Examples/ExamplesApp/ExamplesApp.swift [hunk 2]
```diff
@@ -26,12 +26,17 @@ struct ExamplesApp: App {
...
-         Production deployment of applications built with ArcGIS Maps SDK requires you to license ArcGIS Maps SDK functionality. For more information see https://links.esri.com/arcgis-maps-sdk-license-and-deploy.
+         Use of ArcGIS location services, such as basemaps, geocoding, and routing, requires either user authentication or API key authentication. For more information see https://links.esri.com/arcgis-maps-sdk-security-auth or https://links.esri.com/arcgis-maps-sdk-license-and-deploy.
+         1) User authentication: Allows users with an ArcGIS account to sign into an application and access secure services.
+         2) API key authentication: A long-lived access token that gives your application access to secure services. Go to the [Create an API key](https://developers.arcgis.com/documentation/security-and-authentication/api-key-authentication/tutorials/create-an-api-key/) tutorial to copy the API key access token. To access all services used by this application set the location services privileges to:
+         
+           Location services > Basemaps > Basemaps style service
```
Comment (maryharvey) [inline Examples/ExamplesApp/ExamplesApp.swift:33]: Can you update this to

Location services > Basemaps
Location services > Geocoding
Location services > Routing

This means that the text is correct for both ArcGIS Location Platform accounts and ArcGIS Online accounts. This was a recent change/update. Thanks

File: Examples/ExamplesApp/ExamplesApp.swift [hunk 3]
```diff
@@ -26,12 +26,17 @@ struct ExamplesApp: App {
...
+           Location services > Geocoding > Geocode (not stored)
+           Location services > Routing > Simple routing
+         
+        Production deployment of applications built with the ArcGIS Runtime SDK requires you to license ArcGIS Runtime functionality. For more information see https://links.esri.com/arcgis-runtime-license-and-deploy.
```
Comment (maryharvey) [inline Examples/ExamplesApp/ExamplesApp.swift:37]: built with the ArcGIS Maps SDK for Swift requires that you license your app. For more information see https://developers.arcgis.com/swift/license-and-deployment/.

File: Examples/ExamplesApp/ExamplesApp.swift (hunk 2 above)
Comment (maryharvey) [reply Examples/ExamplesApp/ExamplesApp.swift:33]: I don't know what the toolkit can do. Only request that they set these privileges if they need them.

File: README.md (hunk 1 above)
Comment (maryharvey) [reply README.md:52]: "such as basemap styles, geocoding, and routing services,"

Comment (maryharvey) [reply README.md:52]: "including basemap styles"

Comment (maryharvey) [reply README.md:52]: How about.... "Please see the [Get started guide](https://developers.arcgis.com/swift/get-started/#3-get-an-access-token), [Create an API Key](https://developers.arcgis.com/documentation/security-and-authentication/api-key-authentication/tutorials/create-an-api-key/) and [API Key Authentication](https://main.sites.afd.arcgis.com/swift/security-and-authentication/#api-key-authentication)"

File: README.md [hunk 4]
```diff
@@ -47,9 +47,11 @@ The *ArcGIS Maps SDK for Swift Toolkit* has a *Target SDK* version of *16.0*, me
...
+
+Production deployment of applications built with ArcGIS Maps SDK requires you to license ArcGIS Maps SDK functionality. For more information see https://links.esri.com/arcgis-maps-sdk-license-and-deploy.
```
Comment (maryharvey) [inline README.md:54]: How about this link - https://main.sites.afd.arcgis.com/swift/license-and-deployment/

**Review by maryharvey — COMMENTED** [review]
@dfeinzimer some feedback. thanks.

**Review by mhdostal — APPROVED** [review]

**Comment by dfeinzimer** [comment]
@maryharvey, any further changes on this you'd like to see?

File: README.md [hunk 5]
```diff
@@ -47,9 +47,11 @@ The *ArcGIS Maps SDK for Swift Toolkit* has a *Target SDK* version of *16.0*, me
...
+Use of ArcGIS location services, such as basemap styles, geocoding, and routing services, requires either user authentication or API key authentication. Some of the toolkit components and examples utilize a set of these ready-to-use ArcGIS location services, including basemap styles, and therefore require an API Key to be set in `ExamplesApp.swift`. Please see [Get started](https://developers.arcgis.com/swift/get-started/#3-get-an-access-token), [Create an API Key](https://developers.arcgis.com/documentation/security-and-authentication/api-key-authentication/tutorials/create-an-api-key/) and [API Key Authentication](https://developers.arcgis.com/swift/security-and-authentication/#api-key-authentication) for more information.
+
+Production deployment of applications built with the ArcGIS Maps SDK for Swift requires that you license your app. For more information see https://developers.arcgis.com/swift/license-and-deployment/.
```
Comment (maryharvey) [inline README.md:54]:
```suggestion
Production deployment of applications built with the ArcGIS Maps SDK for Swift requires that you license your app. For more information, see [License and deployment](https://developers.arcgis.com/swift/license-and-deployment).
```

**Review by maryharvey — APPROVED** [review]
@dfeinzimer Looks great and all the links work .. so thanks. One tiny tweak.

**Review by mhdostal — APPROVED** [review]

**Pull request merged**